### PR TITLE
added base convertor program

### PR DESCRIPTION
**Pull request by prithvirajbytes**
Hi, added one basic base convertor system python program which helps to convert the base of number. Please merge and help me contribute open source.

### CODE REVIEW

**Comment by Rishikesh-12** [comment]
hello, @prithvirajbytes  Hactoberfest will be starting from tomorrow so I will merge your pull request tomorrow.
 also I would like you to read the readme once. You need to also contribute to the contributions.md

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -17,4 +17,9 @@ If Table row already has 7 contributors information make a new row-->
   <tr>
     <td align="center"><a href="https://rishikeshmishra.netlify.app"><img src="https://avatars1.githubusercontent.com/u/54947439?s=400&u=db1c8d20adbb31328a878cc95e6467135b9ad144&v=4" width="100px;" alt=""/><br /><sub><b>Rishikesh Mishra</b></sub></a><br /></td>
   </tr>
+</table>
```
Comment (Rishikesh-12) [inline CONTRIBUTORS.md:20]: remove line 19,20,21

Comment (prithvirajbytes) [reply CONTRIBUTORS.md]: removed

Comment (Rishikesh-12) [reply CONTRIBUTORS.md]: great !

**Comment by Rishikesh-12** [comment]
just make a minor change **tomorrow**
add a new blank line anywhere in the file
and then make a pr **tomorrow** so it will be counted.
Okay.

**Comment by prithvirajbytes** [comment]
Dear Rishikesh, hacktober fest has started and counting PR's. Please have a look.

**Comment by navneetsn18** [comment]
@all-contributors please add @prithvirajbytes for code

**Pull request merged**